docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+
+
+### Step 4 : Testing the package
+Now that you have the package built, we need to make sure everything works as expected before publishing to the community.
+
+
+#### Validation using build script.
+You can execute the script inside the `scripts/build.sh` to make sure all the JSON schema conform to their respective schemas and to install any missing libraries. This script is also executed as a precommit hook.
+
+It may throw some errors if there are any unrecognized fields in the package files. Fix those errors and re-execute the command until the build is successful.
+
+Now, we can build the Universe server locally, and run in our DC/OS cluster to test and install our package.
+
+Universe Server is a new component introduced alongside `packagingVersion` `3.0`. In order for Universe to be able to provide packages for many versions of DC/OS at the same time, it is necessary for a server to be responsible for serving the correct set of packages to a cluster based on the cluster's version.
+
+
+#### Build the Universe server
+Build the Universe Server Docker image:
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash
+```
+
+This will create a Docker image `universe-server:time-server` and `docker/server/target/marathon.json` on your local machine.
+
+If you would like to publish the built Docker image to Docker Hub, run:
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash publish
+```
+
+
+#### Run the local Universe server
+Using the `marathon.json` that is created when building Universe Server we can run a Universe Server in our DC/OS cluster which can then be used to install packages.
+
+Run the following commands inside the `server/target` directory to configure DC/OS to use the custom Universe Server (DC/OS 1.8+):
+
+`dcos marathon app add marathon.json`
+
+
+#### Add the Universe repo to DC/OS cluster:
+Now that you have local Universe Server up and running, add it to the cluster's package manager as a repository. You can do this from the GUI or CLI. To achieve this from CLI, execute:
+
+`dcos package repo add --index=0 dev-universe http://universe.marathon.mesos:8085/repo`
+
+
+#### Install the package
+- You can search for your package using something like:
+
+    `dcos package search time`
+- Once you have found your `time-server` package, you can install it onto your cluster using
+
+    `dcos package install time-server`
+- Install the package and if everything works, you have successfully created a package, tested and deployed it! You can check if your package is running at
+
+    `dcos marathon package list`
```

Comment:
Sorry, I mistyped this in my original comment. It should be `dcos package list`, no `marathon`.